examples/source/interactivity-dynamic-content/Paged_List/api.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const express = require('express');
+
+// eslint-disable-next-line new-cap
+const examples = express.Router();
+const MAX_PAGE_COUNT = 5;
+const ITEMS_PER_PAGE = 4;
+
+class ProductListing {
```

Comment:
Defining classes for simple data objects is overkill in javascript. 

```
const productListing = {
  title: ...,
  ...
}
```

is easier to understand in this case.

Reply:
PTAL